control/cli.py

```diff
@@ -226,7 +226,7 @@ def add_namespace(self, args):
     def remove_namespace(self, args):
         """Removes a namespace from a subsystem."""
         req = pb2.remove_namespace_req(subsystem_nqn=args.subnqn,
-                                        nsid=args.nsid)
+                                        nsid=args.nsid, already_locked=False)
```

Comment:
can we use the locked() method instead?

Reply:
Not relevant any more as this parameter was reverted. In any case we couldn't have used locked() here. This parameter meant that the current thread was already holding the lock, not that the lock is held by anyone.